doc/bitcoin-conf.md

```diff
@@ -0,0 +1,11 @@
+# `bitcoin.conf` - Bitcoin configuration file
+
+All command-line options (except for `-conf`) may be specified in a configuration file, and all configuration file options (except for `-includeconf`) may also be specified on the command line. Command-line options override values set in the configuration file and configuration file options override values set in the GUI.
+
+The configuration file is a plain text file and consist of 'setting=value' entries, one per line. Leading and trailing whitespaces are removed. A number sign (`#`) indicates a comment. Comments and empty lines are ignored.
+
+The configuration file is not automatically created; you can create it using your favorite plain-text editor. By default, the configuration file name is `bitcoin.conf` and it is located in the Bitcoin data directory, but both the Bitcoin data directory and the configuration file path may be changed using the `-datadir` and `-conf` command-line arguments.
+
+The `includeconf=<file>` option in the `bitcoin.conf` file can be used to include additional configuration files.
+
+Network specific options can be placed into the sections with headers `[main]` (not `[mainnet]`), `[test]` (not `[testnet]`) and `[regtest]`.
```

Comment:
I believe you can also write network specific options by prefixing the option with `netname.`.

Reply:
yes, though I think in general when writing user documentation, documenting one way that works (and is recommended) is good enough; the conversion to `a.b` might be seen as an implementation detail that will just confuse

but don't care strongly, could add it as extra information